### ISSUE TITLE
duplicate divs with ids (entries) lacking heads without <cont>

### ISSUE THREAD

**Issue by njr2128** [cu-mkp/m-k-manuscript-data#1452]
These are being treated as entries without heads
If continueds are put in, it will resolve the problem but introduce continues/continued in rendering

e.g.,
https://github.com/cu-mkp/m-k-manuscript-data/blob/429227a44e30a6292c2db5d766cf593cfeec73c2/ms-xml/tc/tc_p001r_preTEI.xml#L209

xpath query
> //div[@id][not(head)][not(@continues)][not(@continued)]

**Comment by thuchacz** [comment]
Terry was going to introduce a new attribute from TEI: "part"
Not sure if this got implemented.

**Comment by njr2128** [comment]
@tcatapano - will review and reply here

**Comment by tcatapano** [comment]
@thuchacz : Yes. I should have made an issue for it (or if I did, remembered it). I'll put add it to the schema. I can also add them to the 20 or so divs that apparently need them (i.e., divs with IDs which dont have heads). Does that sound OK?

**Comment by tcatapano** [comment]
@thuchacz : please check the div segments sharing the id p016r_1. The first has categories and the third has continued.

**Comment by thuchacz** [comment]
@tcatapano
I put the categories in the first div segment because it was the first
occurrence of that div id on the page.
I put the "continued" attribute in the last div segment because it renders
at the bottom of the div it appears in. I did not want "continued..." to
show up in all three segments.

Is this problematic?

**Comment by tcatapano** [comment]
@thuchacz: I think all we need to do is put part="y" in the non-first divs to chain all the segments together and avoid having the "continued" show up. Categories **should** go in the first segment so there's no problem.

That said, Im a bit chagined and perplexed that the changes from this commit did not seem to be reflected in the history of this file:

https://github.com/cu-mkp/m-k-manuscript-data/commit/bff8db22f6627602328c023d1939bfbc2f46febe

I thought I put in the part="y" attributes!

**Comment by thuchacz** [comment]
@tcatapano
Okay. So... 2 questions then:
1. are you searching and adding part="y" attributes in some fancy way, or
should I go back and add them manually, now that they're in the schema?
2. from our earlier conversation, I thought the first div also needed a
part="y" attribute, if only to say that it was an initial part of a
multi-part div. Please confirm one way or the other, esp. if I need to
insert these manually.

**Comment by tcatapano** [comment]
@thuchacz: OK. My changes are present in the master branch files.  See:

https://github.com/cu-mkp/m-k-manuscript-data/blob/master/ms-xml/tc/tc_p001r_preTEI.xml

I was looking and some other version of the file. This is a pretty minor issue at this point, I'll take care of the rest. In answer to your questions:

1) Yes, I did a fancy search and replace to put the part="y" 's in.... but
2) You are right, the initial segment of these segmented div's would best also have a part="y". It's not very consequential, however and a low prioirty. You dont have to do anything. Again, I'll take care of it.

I should have documented the decisions made about this issue and its solution when we came up with them. Good lesson to be rememebered (especially by me)
